Tests/UITests/TestApp/OnboardingTestsView.swift

```diff
@@ -15,42 +15,41 @@ struct OnboardingTestsView: View {
     @Binding var onboardingFlowComplete: Bool
     @State var showConditionalView = false
 
-
     var body: some View {
-        OnboardingStack(onboardingFlowComplete: $onboardingFlowComplete) {
+        OnboardingStack(didComplete: $onboardingFlowComplete) {
```

Comment:
Would argue to not use the typealias here but directly use the `ManagedNavigationStack`. Also applies to all docs